js/screenSaver.js

```diff
@@ -0,0 +1,35 @@
+var timelimit=60000;//1 minute
+function screenSaver () {
+	Timer=setTimeout(activate,timelimit);
+	document.addEventListener('click',deactivate,false);
+	document.addEventListener('mousemove',deactivate,false);
+	document.addEventListener('keypress',deactivate,false);
+}
+function deactivate(){
+	clearTimeout(Timer);
+	var parent=document.getElementsByTagName("body");
+	var child=document.getElementById("screen");
+	child.style.display="none";
+	Timer=setTimeout(resume,timelimit);
+}
+function resume(){
+	var image=document.getElementById("screen");
```

Comment:
you have

```
var child=document.getElementById("screen");
```

and then
```
var child=document.getElementById("screen");
```

For me this is confusing, since the file thing is named differently. I suggest using the same name for the same thing.